support/Cargo.toml

```diff
@@ -27,7 +27,7 @@ sp-runtime.workspace = true
 sp-std.workspace = true
 
 # std dependencies
-serde = {workspace = true, optional = true}
+serde = {workspace = true, default-features= false, features=["derive"]}
```

Comment:
`default-features` is already specified in the root TOML, so you can skip that here.

Reply:
True - thanks :raised_hands: [4a5345a](https://github.com/KILTprotocol/kilt-node/pull/559/commits/4a5345af379dfbd0179ab78db4eb6fd87a49deb8)